Twitter Sentiment Analysis.py

```diff
@@ -1,8 +1,8 @@
 import tweepy
-
 import csv
-
 from textblob import TextBlob
+import matplotlib.pyplot as plt
+import seaborn as sns
```

Comment:
Since we are importing another library here we should add the description of it in the README file.

Reply:
Will do